packages/api/tests/api/subscription.test.ts

```diff
@@ -0,0 +1,102 @@
+import { subscription } from '../../src/api/subscription';
+import { User } from '../../src/entities/User';
+import logger from '../../src/logger';
+import { testHandler } from '../testUtils/testHandler';
+
+const loggerSpy = jest.spyOn(logger, 'error').mockImplementation();
+
+const mockUser = {
+  authId: 'testAuthId',
+};
+
+const mockSession = {
+  profile: {
+    id: mockUser.authId,
+  },
+  accessToken: 'accessToken',
+  refreshToken: 'refreshToken',
+};
+
+describe('/subscriptions', () => {
+  beforeEach(async () => {
+    jest.clearAllMocks();
+  });
+
+  it('successfully subscribes a user', async () => {
+    const handler = testHandler(subscription, (req, _res, next) => {
```

Comment:
Because this test bypasses the file that uses `populateUser`, you won't actually need to test that here. Instead you'll need to do something like this:

```suggestion
    const handler = testHandler(subscription, (req, _res, next) => { 
      req.user = mockUser;
      next(); 
    }, 
    (req, _res, next) => {
```

That inserts new middleware into the mock test handler that lets you modify the `req` object like we'll do in the actual middleware